### DISCUSSION TITLE
[Swift] Support for 'step_name' in Fastlane.swift functions?

### DISCUSSION THREAD

**Discussion by mwhrtin** [fastlane/fastlane#21881] [Ideas]
Currently I have not been able to find how I can specify a custom step name when using Fastlane.swift, e.g. when calling `match`, `buildIosApp` or any of the other defined functions. All actions support a step name using the `step_name` argument starting with `fastlane 2.120.0` per the documentation, but Fastlane.swift doesn't define this as an input parameter. 

To workaround this I've copied the functions I want to be able to specify a custom step name with and I've simply added a `stepName` input to them that maps to the `step_name` argument. It works, but I'd prefer if I didn't have to maintain local copies of the functions.

Have I just missed how I can specify a step name with Fastlane.swift, or is this not supported out of the box?

**Comment by ddeveloperuch** [comment]
I'm also interested in the answer to this question.
